roofit/histfactory/src/FlexibleInterpVar.cxx

```diff
@@ -189,15 +189,15 @@ void FlexibleInterpVar::setInterpCode(RooAbsReal& param, int code){
   int index = _paramList.index(&param);
   if(index<0){
       coutE(InputArguments) << "FlexibleInterpVar::setInterpCode ERROR:  " << param.GetName()
-             << " is not in list" << endl ;
-  } else {
-      coutW(InputArguments) << "FlexibleInterpVar::setInterpCode :  " << param.GetName()
-             << " is now " << code << endl ;
-    _interpCode.at(index) = code;
+			    << " is not in list" << endl ;
+  } else if(_interpCode.at(index) != code){
```

Comment:
Are the changes to `FlexibleInterpVar.cxx` really necessary, or was it just temporary for debugging?

Reply:
This is (sort of) needed. Otherwise, the output gets flooded with these printouts whenever importing a HistFactory model.

Reply:
Okay! Just wanted to make sure. I also don't understand why it was a warning to begin with